tests/unit/core/containers/error-simulation/TestSimulateClientError.js

```diff
@@ -20,27 +19,49 @@ describe('SimulateClientError', () => {
   });
 
   function render(props = {}) {
-    return findDOMNode(
-      renderIntoDocument(<SimulateClientErrorBase {...props} />));
+    return shallowUntilTarget(
+      <SimulateClientError {...props} />,
+      SimulateClientErrorBase
+    );
   }
 
   it('lets you trigger an error', () => {
     const root = render();
-    const button = root.querySelector('button');
-    expect(() => Simulate.click(button)).toThrowError(/simulated client error/);
+
+    expect(() => {
+      root.find(Button).at(0).simulate('click');
```

Comment:
Instead of `find(Button).at(0)` I'd rather see it query by class name now that we have two buttons.